### DISCUSSION TITLE
Keyboard shortcut to change layout

### DISCUSSION THREAD

**Discussion by abdullah-if** [OpenBangla/OpenBangla-Keyboard#243] [Ideas]
আমি সাধারনত `ibus-avro` ব্যাবহার করি। কিছুদিন ধরে আমি `প্রভাত` লেয়াউট ব্যাব্যহার করছি, যেটা `xkeyboard-config` দিয়ে থাকে, `ibus` এর মাধ্যমে। প্রভাত লেয়াউটে কিবোর্ড সর্টকাটগুলো কাজ করে না (যেহেতু Ctrl + A != Ctrl + া )। ওপেনবাংলা কিবোর্ডয়ে অভ্র ফনেটিক আর প্রভাত দুইটা লেয়াউটই আছে তাই আমি এটা ব্যাবহার করছি। কিন্তু লেয়াউট বদলাতে প্রতি বার মাউস দিয়ে ঠিক করতে হয়। এটার জন্য কিবোর্ড সর্টকাট দিলে ভালো হয়।

**Comment by mominul** [comment]
Transferred this as an issue #244
